arcgis-ios-sdk-samples/Geometry/Nearest vertex/NearestVertexViewController.swift

```diff
@@ -16,72 +16,76 @@ import UIKit
 import ArcGIS
 
 class NearestVertexViewController: UIViewController {
-    /// The graphics and symbology for the tapped point, the nearest vertex and the nearest coordinate.
-    let tappedLocationSymbol = AGSSimpleMarkerSymbol(style: .X, color: .orange, size: 15)
-    let nearestCoordinateSymbol = AGSSimpleMarkerSymbol(style: .diamond, color: .red, size: 10)
-    let nearestVertexSymbol = AGSSimpleMarkerSymbol(style: .circle, color: .blue, size: 15)
+    // MARK: Properties
     
-    /// The symbology for the example polygon area.
-    let polygonFillSymbol = AGSSimpleFillSymbol(
-        style: .forwardDiagonal,
-        color: .green,
-        outline: AGSSimpleLineSymbol(style: .solid, color: .green, width: 2)
-    )
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap(basemapStyle: .arcGISTopographic)
+            mapView.graphicsOverlays.add(graphicsOverlay)
+            mapView.setViewpointCenter(polygon.extent.center, scale: 1e8)
+            mapView.touchDelegate = self
+            mapView.callout.isAccessoryButtonHidden = true
+        }
+    }
     
-    /// The graphics overlay for the polygon and points.
+    /// The graphics overlay for the polygon and point graphics.
     let graphicsOverlay = AGSGraphicsOverlay()
     
-    /// A formatter to convert units for distance.
+    /// The example polygon geometry.
+    let polygon: AGSPolygon
+    
+    /// The graphic for the polygon.
+    let polygonGraphic: AGSGraphic
+    /// The graphic for the tapped location point.
+    let tappedLocationGraphic: AGSGraphic
+    /// The graphic for the nearest coordinate point.
+    let nearestCoordinateGraphic: AGSGraphic
+    /// The graphic for the nearest vertex point.
+    let nearestVertexGraphic: AGSGraphic
```

Comment:
Consider initializing these inline as well.

```swift
let tappedLocationGraphic: AGSGraphic = {
    let symbol = AGSSimpleMarkerSymbol(style: .X, color: .orange, size: 15)
    return AGSGraphic(geometry: nil, symbol: symbol)
}()
let nearestCoordinateGraphic: AGSGraphic = {
    let symbol = AGSSimpleMarkerSymbol(style: .diamond, color: .red, size: 10)
    return AGSGraphic(geometry: nil, symbol: symbol)
}()
let nearestVertexGraphic: AGSGraphic = {
    let symbol = AGSSimpleMarkerSymbol(style: .circle, color: .blue, size: 15)
    return AGSGraphic(geometry: nil, symbol: symbol)
}()
```

With all of the properties initialized inline, the implementation of `init?(coder:)` isn't needed.